### ISSUE TITLE
After opening a chat chat input text entry field is stealing the focus

### ISSUE THREAD

**Issue by pvagner** [deltachat/deltachat-desktop#5097]
- Operating System (Linux/Mac/Windows/iOS/Android): Arch Linux
- Delta Chat Version: 1.58.1 379d77f6d
- Expected behavior: I'd like to use tab and shift+tab to navigate the UI e.g. search field, profile list, toolbars, chat log, chat input and so on.
- Actual behavior: When a chat has been opened at least once chat input text entry field is being focused all the time instead.
- Steps to reproduce the problem:

  * Launch deltachat-tauri
  * Notice the last active chat conversation opens automatically. The chat input text entry field has the focus so you can start to type your message.
  * Now press ctrl+f keyboard shortcut to move focus to the search entry field.
  * Now try to navigate with tab or shift+tab and notice how you are unable to focus other focusable elements besides chat input.

Please note I have built both tauri and electron editions of deltachat-desktop on my own. I can reliably reproduce this with deltachat-tauri and I can't reproduce this with deltachat-desktop thus I assume this might be tauri specific.

**Comment by WofWca** [comment]
Thanks for the report! I would say this is probably a sub-issue of https://github.com/deltachat/deltachat-desktop/issues/4590.

So, what you're saying is that the focus never leaves the message input or the search input at all? What happens when you press Tab? The input gets focused again?
This is not the case on Windows at least.

**Comment by pvagner** [comment]
@WofWca  Thanks for acknowledging. I apologize, I've missed #4590.

My precise steps are like this:

* Launch delta-tauri
* The window opens and chat input receives the focus. Screen reader reports "Write a message - my contact partner name - Deltachat".
* I press tab to move focus forwards or press shift+tab to move focus backwards and nothing happens. I can press shift+tab multiple times and nothing happens. After pressing shift+tab like 10 times I can begin typing to verify chat input is still in focus.
* In order to better showcase the issue I can press ctrl+f which will correctly move focus to the search input.
* With search input focused pressing tab or shift+tab moves focus to the chat input. Thus other than using advanced screen reader features like object navigation or flat review I have no other way on how to navigate the tauri edition of the app.
*
